webui/react/src/components/kit/LineChart.tsx

```diff
@@ -323,18 +337,34 @@ export const ChartGrid: React.FC<GroupProps> = React.memo(
             <XAxisFilter options={xAxisOptions} value={xAxis} onChange={onXAxisChange} />
```

Comment:
The ChartGrid's `ScaleSelect` above should also be hidden when empty or isLoading